internal/devices/create.go

```diff
@@ -219,7 +231,7 @@ func (c *Client) Create() *cobra.Command {
 	createDeviceCmd.Flags().IntVarP(&publicIPv4SubnetSize, "public-ipv4-subnet-size", "S", 0, "Size of the public IPv4 subnet.")
 	createDeviceCmd.Flags().StringVarP(&hardwareReservationID, "hardware-reservation-id", "r", "", "The UUID of a hardware reservation, if you are provisioning a server from your reserved hardware.")
 	createDeviceCmd.Flags().StringVarP(&billingCycle, "billing-cycle", "b", "hourly", "Billing cycle ")
-	createDeviceCmd.Flags().BoolVarP(&alwaysPXE, "always-pxe", "a", false, "Sets whether the device always PXE boots on reboot.")
+	createDeviceCmd.Flags().StringVarP(&alwaysPxe, "alwaysPxe", "a", "", "Sets whether the device always PXE boots on reboot.")
```

Comment:
Let's restore the original name to keep the existing naming convention and not change the interface.